lib/rspecq/worker.rb

```diff
@@ -15,6 +15,12 @@ class Worker
     # will be split and scheduled on a per-example basis.
     attr_accessor :file_split_threshold
 
+    # Retry failed examples up to N times (with N being the supplied value)
+    # before considering them legit failures
+    # 
+    # Defaults to the value set in DEFAULT_MAX_REQUEUES
```

Comment:
Like discussed in the comments, let's get rid of this constant here to further simplify things.

> Defaults to 3.